scripts/recan_bundle_tdr.py

```diff
@@ -108,27 +109,26 @@ def content_length(content: JSON) -> int:
     return len(json.dumps(content).encode('UTF-8'))
 
 
-namespace = uuid.UUID('5c5284e7-b739-49cf-866a-15db5a049fd2')
-random_deterministic_uuids: Iterator[str] = (
-    str(uuid.uuid5(namespace, str(i)))
-    for i in itertools.count()
-)
-
 epoch = datetime.utcfromtimestamp(0)
 incrementing_versions: Iterator[datetime] = (
     epoch + timedelta(seconds=i)
     for i in itertools.count()
 )
 
 
-def random_uuid() -> str:
-    return next(random_deterministic_uuids)
-
-
 def incrementing_version() -> str:
     return tdr.Plugin.format_version(next(incrementing_versions))
 
 
+namespace_counts: Dict[uuid.UUID, int] = {}
```

Comment:
What you wrote is perfectly functional, but the ideal data structure already exists in the standard lib https://docs.python.org/3/library/collections.html#collections.Counter